client/src/pages/Dashboard/Dashboard.tsx

```diff
@@ -51,7 +55,7 @@ export default function Dashboard(): JSX.Element {
     <Box>
```

Comment:
@eng-dallavecchia I made some changes to  `Dashboard.tsx` in #63 to ensure the page isn't handling data it doesn't need; i.e., list of boards, as that should live in the `OptionsDrawer` instead!

Reply:
Thanks!